### PR TITLE
fix: Fixed the issue that shift+enter does not work

### PR DESCRIPTION
**Pull request by JimLiu**
Fixes Issue: https://github.com/devtools-html/debugger.html/pull/6906#issuecomment-418409026

Enter and Shift + Enter

Project search and highlight

### CODE REVIEW

File: src/components/ProjectSearch.js [hunk 1]
```diff
@@ -147,12 +147,12 @@ export class ProjectSearch extends Component<Props, State> {
...
-      getEditor(),
-      matchItem.line,
-      matchItem.column
-    );
+    // this.props.doSearchForHighlight(
+    //   this.state.inputValue,
+    //   getEditor(),
+    //   matchItem.line,
+    //   matchItem.column
+    // );
```
Comment (jasonLaster) [inline src/components/ProjectSearch.js:155]: why isn't this needed?

**Review by jasonLaster — COMMENTED** [review]
Looks good to me

**Comment by jasonLaster** [comment]
hmm, i'd prefer not to have the right-sidebar search results. Atleast for now as we don't currently have it.

File: src/components/ProjectSearch.js (hunk 1 above)
Comment (JimLiu) [reply src/components/ProjectSearch.js:155]: Hi @jasonLaster 
We still need it, I just changed the place to 
https://github.com/rebornix/debugger.html/pull/2/files#diff-cfc118adfafc73d553d9d93a28eb0a15R417

**Comment by rebornix** [comment]
We don't have an option for disabling the match highlights in overview ruler (I'm surprised that we don't have that ;( ). Either we add this to Monaco or change the color for the highlights, but I prefer the former solution.

**Comment by rebornix** [comment]
@JimLiu thanks! The search works great now IMO.

**Pull request merged**